lib/ansible/modules/cloud/hcloud/hcloud_server.py

```diff
@@ -0,0 +1,408 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Hetzner Cloud GmbH <info@hetzner-cloud.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = """
+---
+module: hcloud_server
+
+short_description: Create and manage cloud servers on the Hetzner Cloud
+
+version_added: "2.8"
+
+description:
+    - "Create, update and manage cloud servers on the Hetzner Cloud."
+
+author:
+    - Lukas Kaemmerling (@lkaemmerling)
+
+options:
+    id:
+        description:
+            - The Server ID
```

Comment:
We tend to have full descriptions, so that the meaning is more clear to the end-user.
The first line is a description of the parameter. Subsequent lines clarify the value, its use or dependencies.
All descriptions (except the short_description) should have a trailing dot, so please add those everywhere.
```suggestion
            - The ID of the Hetzner server to manage.
```